src/history.js

```diff
@@ -115,9 +115,13 @@ function Slides({ commits, slideLines }) {
       }
     };
   });
+  const mouseWheelEvent = e => {
+    e.deltaY > 0 ? nextSlide() : prevSlide();
```

Comment:
Also, instead of moving between commits it would be nice to scroll more gradually. Something like" `e => { selectCommit(currentIndex + (e.deltaX + e.deltaY) / 100); }`.